docs/backup/new-backup-strategy.md

```diff
@@ -0,0 +1,164 @@
+# Kyma Backup Strategy
+
+User load on a Kyma cluster typically consists of various Kubernetes objects and volumes. Kyma relies on the backing cloud provider for periodic backups of the Kubernetes objects. That's why it does not require the user to do any manual settings to take backups.
+
+For example, Gardener uses etcd as the Kubernetes' backing store for all cluster data. That is, all Kubernetes objects are stored on etcd. Gardener has perodic jobs to take major and minor backups of etcd database. A major backup including all the resources takes place every hour, and each minor backups including only the changes inbetween take place every five minutes. In case of a problem on etcd database, Gardener automatically restores the Kubernetes cluster using the latest backup.
```

Comment:
I think we should use the term snapshot here when talking about the gardener mechanism, not sure though